agent/bench-scripts/tests/pbench-fio/test-04.txt

```diff
@@ -24,14 +24,19 @@ size=4096M
 numjobs=1
 
 running fio job: /var/tmp/pbench-test-bench/pbench-agent/fio_test-04_1900.01.01T00.00.00/1-rw-4KiB/fio.job (sample1)
+[warn][1900-01-01T00:00:00.000000] log_hist_msec specified in job file but failed to find any log files (.../clients/localhost/fio_clat_hist.*.log*) to process for histograms
```

Comment:
So awkward to put together all the pieces of the test command structure to figure out whether these gold file changes are good. I'm assuming this must indicate a single-host fio where we were previously not looking for these logs; but there seem to be rather a lot of them. And are there (single host) test cases were we do have the logs to validate that they're processed now?

Reply:
This was an attempt to make a minimal change without opening a can of worms.  Unfortunately, it failed.

By making this change, almost all of our fio tests have latency profiles that were not being computed, and so we have this deluge of changes in the test environment.